### PR TITLE
Update deployment workflow in READMEs 

### PR DESCRIPTION
**Pull request by ivarpruijn**
- Deployment workflow updates
- Point to main README from /docs README instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,112 +37,116 @@ Edit or add the Markdown files under `/blog`. Prepend the filename with a date,
...
-This command starts a local development server and open up a browser window. Most changes are reflected live 
-without having to restart the server.
 
-## Build the website
+### Testing a branch on staging
+To test a branch on staging, first of course check it out (`git checkout <YOUR_BRANCH>`), make sure to first 
+provision the appropriate environment variables for SFTP, and then run:
+```bash
+# pull any changes, and build the deployable docker images for both website and docs for staging & production.
+git pull && make build-docker-build-image && make build-docker-deploy-image
```
Comment (borft) [inline README.md:59]: you can simply do `make build-docker-build-image build-docker-deploy-image` in  go.

File: README.md [hunk 2]
```diff
@@ -31,112 +37,116 @@ Edit or add the Markdown files under `/blog`. Prepend the filename with a date,
...
-## Building the docker image
+### Test a runnable docker image
 
-It's also possible to build a docker image, that can run the website + docs, using a built-in instance of apache. Building
-this is fairly easy, using `make`:
+It's also possible to make a build of the site & docs that can run it with a built-in instance of apache:
 ```bash
-# this builds a runnable docker image, using the latest dockerized build
-make build-docker-website-image
+make build-docker-build-image && make build-docker-website-image
```
Comment (borft) [inline README.md:90]: this will change when https://github.com/objectiv/objectiv.io/pull/303 is merged

**Review by borft — APPROVED** [review]
a few comments, but overall looks good to me

File: README.md (hunk 2 above)
Comment (ivarpruijn) [reply README.md]: Ah sure, let's update this when we merge that PR then.

**Pull request merged**